x-pack/plugins/code/public/components/main/content.tsx

```diff
@@ -184,12 +169,30 @@ class CodeContent extends React.PureComponent<Props> {
       currentTree &&
       (currentTree.type === FileTreeItemType.File || currentTree.type === FileTreeItemType.Link)
     ) {
+      const { isUnsupported, isOversize, isImage } = this.props.file!;
+
+      const buttonOptions = [
+        {
+          id: ButtonOption.Code,
```

Comment:
let's use “content” instead of “code”, in the case of markdown, binary, image

keep "code" when it is actually code

Reply:
Ok